Create ui-c8y-1022-27-0-wildcard-search-replacing-the-current-full-text-search.md (#3700)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-27-0-wildcard-search-replacing-the-current-full-text-search.md

---
date: ""

> title: Replace full-text search with wildcard search for improved results

Reviewer: ```suggestion
title: Improved search results by replacing full-text search with wildcard search
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64410
version: 1022.27.0
---

> The previous full-text search often provided inaccurate results. We have replaced the full-text search with a wildcard search that checks if the name contains the search term. Searching for external IDs, descriptions, and types is no longer possible. If you require this functionality, please contact support or disable the 'ui.search.wildcard' feature flag.

Reviewer 2: ```suggestion
To enhance the search experience for assets and devices, we have replaced the legacy full-text search with a more accurate wildcard-based search. This update provides more predictable and relevant results by matching names that contain your search term. As part of this change, searching by external IDs, descriptions, and types is no longer supported. If you require full-text search functionality, please contact support or disable the 'ui.search.wildcard' feature flag.
```

> To enhance the search experience for assets and devices, we have replaced the legacy full-text search with a more accurate wildcard-based search. This update provides more predictable and relevant results by matching names that contain your search term. As part of this change, searching by external IDs, descriptions, and types is no longer supported. If you require full-text search functionality, please contact support or disable the 'ui.search.wildcard' feature flag.

Reviewer: ```suggestion
To enhance the search experience for assets and devices, the legacy full-text search has been replaced with a more accurate wildcard-based search. This update provides more predictable and relevant results by matching names that contain your search term. As part of this change, searching by external IDs, descriptions, and types is no longer supported. If you require full-text search functionality, please contact support or disable the 'ui.search.wildcard' feature flag.
```
